### DISCUSSION TITLE
error: building cpprestsdk:x64-linux failed with: BUILD_FAILED

### DISCUSSION THREAD

**Discussion by silver-mian** [microsoft/vcpkg#36550] [General]
sea@ubuntu:~$ vcpkg install cpprestsdk
Computing installation plan...
The following packages will be built and installed:
    cpprestsdk[compression,core]:x64-linux@2.10.19
Detecting compiler hash for triplet x64-linux...
Restored 0 package(s) from /home/sea/.cache/vcpkg/archives in 1.4 ms. Use --debug to see more details.
Installing 1/1 cpprestsdk[compression,core]:x64-linux@2.10.19...
Building cpprestsdk[compression,core]:x64-linux@2.10.19...
-- Using cached Microsoft-cpprestsdk-411a109150b270f23c8c97fa4ec9a0a4a98cdecf.tar.gz.
-- Cleaning sources at /home/sea/Downloads/vcpkg/buildtrees/cpprestsdk/src/a4a98cdecf-716fde1512.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/sea/Downloads/vcpkg/downloads/Microsoft-cpprestsdk-411a109150b270f23c8c97fa4ec9a0a4a98cdecf.tar.gz
-- Applying patch fix-find-openssl.patch
-- Applying patch fix_narrowing.patch
-- Applying patch fix-uwp.patch
-- Using source at /home/sea/Downloads/vcpkg/buildtrees/cpprestsdk/src/a4a98cdecf-716fde1512.clean
-- Configuring x64-linux
CMake Warning at installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:318 (message):
  The following variables are not used in CMakeLists.txt:

      WEBSOCKETPP_CONFIG
      WEBSOCKETPP_CONFIG_VERSION

  Please recheck them and remove the unnecessary options from the
  `vcpkg_cmake_configure` call.

  If these options should still be passed for whatever reason, please use the
  `MAYBE_UNUSED_VARIABLES` argument.
Call Stack (most recent call first):
  ports/cpprestsdk/portfile.cmake:33 (vcpkg_cmake_configure)
  scripts/ports.cmake:170 (include)

-- Building x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /home/sea/Downloads/vcpkg/downloads/tools/cmake-3.27.1-linux/cmake-3.27.1-linux-x86_64/bin/cmake --build . --config Debug --target install -- -v -j17
    Working Directory: /home/sea/Downloads/vcpkg/buildtrees/cpprestsdk/x64-linux-dbg
    See logs for more information:
      /home/sea/Downloads/vcpkg/buildtrees/cpprestsdk/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/cpprestsdk/portfile.cmake:48 (vcpkg_cmake_install)
  scripts/ports.cmake:170 (include)

error: building cpprestsdk:x64-linux failed with: BUILD_FAILED
Elapsed time to handle cpprestsdk:x64-linux: 6.5 min
Please ensure you're using the latest port files with `git pull` and `vcpkg update`.
Then check for known issues at:
  https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+cpprestsdk
You can submit a new issue at:
  https://github.com/microsoft/vcpkg/issues/new?title=[cpprestsdk]+Build+error+on+x64-linux&body=Copy+issue+body+from+%2Fhome%2Fsea%2FDownloads%2Fvcpkg%2Finstalled%2Fvcpkg%2Fissue_body.md

**Comment by silver-mian** [comment]
Could you please help me, big shot

**Reply by dg0yt** [reply]
See logs for more information:
/home/sea/Downloads/vcpkg/buildtrees/cpprestsdk/install-x64-linux-dbg-out.log

**Reply by silver-mian** [reply]
[install-x64-linux-dbg-out.log](https://github.com/microsoft/vcpkg/files/14155565/install-x64-linux-dbg-out.log)
thanks

**Reply by dg0yt** [reply]
So which error message do you see in the log? What is your conclusion?

**Reply by silver-mian** [reply]
Sorry, my skills are not good and I cannot understand the log files
